### PR TITLE
feat(widgetbook): :zap: added text knob multiline

### PR DESCRIPTION
**Pull request by mulieriq**
### TextKnob
- Added multiline attribute to text knob 
- Added test for `text knob multiline.`

### List of issues which are fixed by the PR
*You must list at least one issue.*

### Screenshots
*If applicable, add screenshots to help explain the changes.*

### Checklist

- [ ] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making].
- [ ] All existing and new tests are passing.

If you need help, consider asking for advice on [Discord].

[CLA]: https://docs.google.com/forms/d/e/1FAIpQLScuRfjUzENsLsmQgqZlGLxMKbFi7zuXoPARyXytoyQrq7ntUw/viewform
[Discord]: https://discord.com/invite/zT4AMStAJA

### CODE REVIEW

File: packages/widgetbook/lib/src/knobs/text_knob.dart [hunk 1]
```diff
@@ -5,18 +5,16 @@ import 'package:widgetbook/src/knobs/nullable_checkbox.dart';
...
-          description: description,
-          value: value,
-        );
-
+    required super.label,
+    super.description,
+    required super.value,
+    required this.multiline,
+  });
+  bool multiline;
```
Comment (jenshor) [inline packages/widgetbook/lib/src/knobs/text_knob.dart:13]: Can we make this `final`?

**Review by jenshor — COMMENTED** [review]
Except the one comment, this looks good to me 👍🏻

File: packages/widgetbook/lib/src/knobs/text_knob.dart (hunk 1 above)
Comment (mulieriq) [reply packages/widgetbook/lib/src/knobs/text_knob.dart:13]: sure

**Review by jenshor — APPROVED** [review]
LGTM 🎉

**Pull request merged**